src/QAT/qat/purr/backends/realtime_chip_simulator.py

```diff
@@ -644,21 +644,13 @@ def get_default_RTCS_hardware(repeats=1000, rotating_frame=True):
 
 
 class RealtimeSimHardwareModel(QuantumHardwareModel):
-    def __init__(
-        self,
-        *args,
-        engine_types: List[QuantumExecutionEngine] = None,
-        builder_types: List[QuantumInstructionBuilder] = None,
-        **kwargs,
-    ):
-        super().__init__(
-            *args,
-            engine_types=engine_types or [RealtimeChipSimEngine],
-            builder_types=builder_types or [QuantumInstructionBuilder],
-            **kwargs,
-        )
+    def __init__(self, *args, **kwargs):
```

Comment:
This dosen't need to exist at all if you're not doing anything in the constructor.